source/includes/inkling2-reference/_algorithms.html.md

```diff
@@ -40,60 +40,36 @@ The algorithm clause is for advanced users who want to experiment with changing
 | ConvolutionLayers: [ConvolutionLayerInfo] <br><br> (`ConvolutionLayers: {XSize: 8, YSize: 8, XStride: 4, YStride: 4 FilterCount: 32}”`) | An array of structures that define the size, stride and (optionally) the filter count for each convolution layer. |
 | CompressionLayers: [HiddenLayerInfo] <br><br> (`CompressionLayers: [{Size: 20, Activation: "tanh"}, {Size: 10, Activation: "softmax"}]`) | If specified, will add one or more fully-connected hidden layers after the convolutional layers to compress the representation before incorporating other features. The format is the same as the HiddenLayerInfo described above. |
 | ConvolutionLimitDimension: string <br> (`ConvolutionLimitDimension: “x”`) | A string that represents which dimension of the input shape to be used as the convolutional filter sizes. Must be one of: "max", "min", "x", "y", or "none". By default, it’s "none", which uses the original input shapes as the filter sizes. "max" uses the maximum of x and y inputs as the filter size. "min" uses the minimum of x and y input. "x" uses the x dimension input shape as the filter size. "y" uses the y dimension input shape. (The example uses the input x dimension shape as the convolutional filter size.) |
 
 
-### DQN-Specific Parameters
-
-In addition to the first (shared) table of parameters, DQN also supports four additional parameters.
-
-| Parameter (Example use)      | Description |
-| -                            | -           |
-| ExplorationDecay: float <br> (`ExplorationDecay: 0.00001`) | The exploration decay in DQN. |
-| SamplesPerTrain: int <br> (`SamplesPerTrain: 10`) | The number of samples (iterations) collected before training. |
-| QLearningRate: float <br> (`QLearningRate: 0.0001`) | The learning rate for training the Q network. |
-| DiscountRate: float <br> (`DiscountRate: 0.99`) | The discount rate for the Bellman Equation. |
-
-### DDPG-Specific and DiscreteDDPG Parameters
+### APEX-Specific Parameters
 
-In addition to the first (shared) table of parameters, DDPP and DiscreteDDPG support three additional parameters.
+In addition to the first (shared) table of parameters, APEX also supports these additional parameters.
 
 | Parameter (Example use)      | Description |
 | -                            | -           |
-| MuLearningRate: float <br> (`MuLearningRate: 0.0001`) | The learning rate for training the Mu network. |
+| HiddenLayers: [HiddenLayerInfo] <br><br> (`HiddenLayers: [{Size: 400, Activation: "relu"}, {Size: 300, Activation: "tanh"}]`) | An array of structures that define the size and (optionally) the activation function for each hidden layer. Sizes must be positive integers, and activation functions must be one of "linear", "tanh", "relu", "logistic", "softmax", "elu", or "default". |
```

Comment:
I'm surprised to see HiddenLayers move to the APEX-specific section. Is that correct? (@AdityaGudimella)

Reply:
Hidden layer sizes and activations are available for both PPO and Apex. They are not Apex specific.

Reply:
Hmm, the documentation correctly reflects what the inkling v2 compiler allows, and that was implemented according to a spec that I reviewed with Matineh. I propose that we document the currently-implemented behavior (which doesn't expose HiddenLayers for PPO), and we open a new ticket to add that functionality and update the docs at some later time.

Reply:
👍 @erictraut, will you make the ticket?